test/e2e/velero_utils.go

```diff
@@ -41,20 +41,56 @@ import (
 	veleroexec "github.com/vmware-tanzu/velero/pkg/util/exec"
 )
 
-func getProviderPlugins(providerName string) []string {
-	// TODO: make plugin images configurable
-	switch providerName {
-	case "aws":
-		return []string{"velero/velero-plugin-for-aws:v1.2.1"}
-	case "azure":
-		return []string{"velero/velero-plugin-for-microsoft-azure:v1.2.0"}
-	case "vsphere":
-		return []string{"velero/velero-plugin-for-aws:v1.2.1", "vsphereveleroplugin/velero-plugin-for-vsphere:v1.1.1"}
-	case "gcp":
-		return []string{"velero/velero-plugin-for-gcp:v1.2.1"}
-	default:
-		panic(fmt.Errorf("unknown provider name: %s", providerName))
+var latestPluginsMap = map[string][]string{
+	"aws":     {"velero/velero-plugin-for-aws:v1.3.0-rc.1"},
```

Comment:
Let's use the image with tag `main` as the latest version.
As vSphere plugin image has no `main` tag image, keep it as `v1.1.1`